src/content/guides/index.md

```diff
@@ -4,24 +4,38 @@ description: Learn to use the Dart language and libraries.
 toc: false
 ---
 
-Welcome to the Dart documentation!
-For a list of changes to this site—new pages, new guidelines, and more—see
-the [What's new][] page.
-
-[What's new]: /guides/whats-new
```

Comment:
Why are we taking this paragraph out?

Reply:
Ah. Originally, I had replaced it with another card. I can still do that. I'll see what feedback I get from @sethladd first.